modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js

```diff
@@ -89,76 +185,77 @@ const SetupAnalyticsCloudPage = ({
 			}}
 		>
 			<FieldArray
-				name="activations.incidentReportContact"
-				render={({pop, push}) => (
-					<>
-						<ClayForm.Group className="mb-0">
-							<ClayForm.Group className="mb-0">
-								<Input
-									groupStyle="pb-1"
-									helper="This user will create and manage the Analytics Cloud Workspace and must have a liferay.com account. The owner Email can be updated vis Support ticket if needed."
-									label="Owner Email"
-									name="activations.ownerEmail"
-									placeholder="user@company.com"
-									required
-									type="email"
-								/>
-
-								<Input
-									groupStyle="pb-1"
-									helper="Lowercase letters and numbers only. Project IDs cannot be changed."
-									label="Workspace Name"
-									name="activations.workspaceName"
-									placeholder="superbank1"
-									required
-									type="text"
-									validations={[
-										(value) => maxLength(value, 255),
-										(value) => isLowercaseAndNumbers(value),
-									]}
```

Comment:
Maybe you should put this value in a variable like `MAX_LENGTH = 255.`, what do you think?

Reply:
Hey, Joana!! Nice, it's done!